lua/nvim-tree/actions/copy-paste.lua

```diff
@@ -211,7 +211,6 @@ function M.print_clipboard()
   return a.nvim_out_write(table.concat(content, "\n") .. "\n")
 end
 
-local function copy_to_clipboard(content)
```

Comment:
Was this an intentional change?

Reply:
I don't have touch that line. It seems the same for me on this branch

Reply:
Strange, I can see it in the "files changed" now. Probably a github diff bug...

Reply:
i think it's not a bug, the diff is present and needs to be removed (after merging master branch).